test/nlp/test_default.py

```diff
@@ -0,0 +1,81 @@
+from utils import get_toy_data_seqclassification, get_automl_settings
+
+
+def test_starting_point_not_in_search_space():
+    from flaml import AutoML
+
+    """
+        test starting_points located outside of the search space, and custom_hp is not set
+    """
+    X_train, y_train, X_val, y_val, _ = get_toy_data_seqclassification()
+
+    automl = AutoML()
+    automl_settings = get_automl_settings(estimator_name="transformer")
+
+    automl_settings["starting_points"] = {"transformer": [{"learning_rate": 2e-3}]}
```

Comment:
need to verify whether the start point is indeed removed.

Reply:
I did, it was removed.